core/serialisation/data_serialiser.py

```diff
@@ -306,6 +309,25 @@ class Meta:
     organisation_name = auto_field(model=Organisation, data_key="OrganisationName")
 
 
+class ProgrammeManagementSchema(SQLAlchemySchema):
+    """Serialise ProgrammeManagement data"""
+
+    class Meta:
+        model = ProgrammeManagement
+
+    submission_id = auto_field(model=Submission, data_key="SubmissionID")
+    programme_id = auto_field(model=Programme, data_key="ProgrammeID")
+    payment_type = fields.String(attribute="data_blob.payment_type", data_key="PaymentType")
+    spend_for_reporting_period = fields.Integer(
```

Comment:
For other numerical fields we're serialising out of the data blob we use ```fields.Number```. I'm not sure if there's a meaningful difference, but we should probably standardise or specify why different?

Reply:
good spot. I'll check the difference in the docs and choose one method for everything

Reply:
Changed this one to Float. Will fix the other existing ones in a separate PR